tests/kafkatest/sanity_checks/test_console_consumer.py

```diff
@@ -45,11 +45,14 @@ def setUp(self):
         self.zk.start()
 
     @parametrize(security_protocol='PLAINTEXT', new_consumer=False)
+    @parametrize(security_protocol='SASL_SSL', sasl_mechanism='PLAIN')
```

Comment:
@ijuma I thought it is useful to have one sanity test with SASL/PLAIN that is simple to run, hence added a consumer test. Since the replication tests run both producer and consumer anyway, perhaps this is sufficient? `test_verifiable_producer.py` is currently run only with PLAINTEXT.